src/sparsezoo/model/model.py

```diff
@@ -78,12 +79,14 @@ def __init__(self, source: str, download_path: Optional[str] = None):
 
         if self.source.startswith(ZOO_STUB_PREFIX):
             # initializing the files and params from the stub
-            files, path, url = self.initialize_model_from_stub(self.source)
+            _model_args = self.initialize_model_from_stub(self.source)
+            files, path, url, validation_results, compressed_size = _model_args
             if download_path is not None:
                 path = download_path  # overwrite cache path with user input
         else:
             # initializing the model from the path
             files, path, url = self.initialize_model_from_directory(self.source)
+            validation_results = compressed_size = None
```

Comment:
nit: I'd prefer `validation_results, compressed_size = None, None

Reply:
Is it not an identical expression to the one that @rahul-tuli uses? Just wondering

Reply:
For this case yes it's identical, but might not always be